README.md

```diff
@@ -1,1134 +1,233 @@
-Play provides predictable and minimal resource consumption for highly-scalable applications. 
-The Api-First-Hand plugin takes Swagger API definitions and treats them as the single source of truth of your REST services.
-
-Api-First-Hand supports round-trip regeneration and compilation of:
-
-- Play routes definitions (managed).
-- Swagger domain model definitions and parameters onto Scala case classes (managed).
-- Swagger domain model constraints onto Play validations (managed).
-- Generators for random test data generation of parameter values (managed).
-- Unit tests for validating your service at the API boundary (managed).
-- Swagger path definitions onto skeletons for Play controller implementations (unmanaged).
-
-In the list above, "(managed)" means that the code is managed by sbt. The code is not controlled 
-and altered by you, the programmer of the REST service. The plugin takes your Swagger API definition as the single 
-source of truth and regenerates these code parts in a consistent manner.
-You'll instead be focusing on implementing the service business logic in an (unmanaged) Play controller class 
-that is generated once. Subsequent regenerations keep the code that you have added, either by commenting out the 
-parts that are no longer valid, or by adding parts that are needed because you have made a change to the API.
-
-Manual generation and compilation of:
-
-- Security extractors
-- Unmarshallers for custom content types 
+    * `controllers/Swagger.scala`: a backend side of the Swagger UI
+    * `generated_controllers/example.yaml.scala`: a dummy implementation of the example controller that's (re)generated if deleted
+    * `security/example.yaml.scala`: a marshaller for OAuth2 tokens. Will not be regenerated until either deleted or renamed; and then explicitly requested by issuing a `playScalaSecurity` command.
+ 
+## Getting Started with API-First-Hand
 
-is supported in the case if
+### Creating Your Play Application with the Plugin
 
-1. No security extractor or unmarshaller with the same name already exists
-2. The developer issues `playScalaSecurity` or `playScalaMarshallers` sbt command 
+- Use the Activator template, for example: ```activator new hello-world api-first-hand```
+- Look at the `project/plugins.sbt` of your generated project and add the required plugins and resolvers there
+- Do the same for `build.sbt`
+- Put a Swagger specification with a ```.yaml``` or ```.json``` extension into the ```conf``` directory
+- Add a specification link (`->`) to the Play's routes file
 
-## Run Your Application
+### Running Your Application with the Plugin
 
-Before we go any further, let's run the application.
+Now let's run your application with the plugin:
 
 - Open a shell and `cd` into your service project directory.
-- Start `sbt` and `run` the service.
+- Start `sbt` and `run` the service. This creates a folder on your local machine, using the Activator template.
```

Comment:
No, this is not. At this moment an activator template is already present and user runs `sbt` from inside of it. The `run` creates a lot of stuff mentioned in the 'Plugin Features'